### PR TITLE
{lib}[GCCcore/10.3.0] libpsl 0.21.1

### PR DESCRIPTION
**Pull request by casparvl**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by casparvl** [comment]
`libpsl` wouldn't build on one of my systems due to a missing `libunistring`. Added that as dependency now.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.4, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/ee734dbefb8e05333c2785dad069de5f for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
software1.lisa.surfsara.nl - Linux debian 10.13, x86_64, Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz, Python 3.7.3
See https://gist.github.com/8b37ff8d01427fc6a9d36fb221c539ac for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node109.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e4ba4c5a8f1432958f4a0ebdc395f00b for a full test report.

**Comment by jfgrimm** [comment]
@casparvl we should also add a dependency on `ICU`:

```
$ ldd $EBROOTLIBPSL/lib/libpsl.so | grep libicu
	libicuuc.so.50 => /lib64/libicuuc.so.50 (0x00007fa1ab6c6000)
	libicudata.so.50 => /lib64/libicudata.so.50 (0x00007fa1aa0f3000)

```

File: easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -19,6 +19,10 @@ builddependencies = [
     ('CMake', '3.20.1'),
 ]
 
+dependencies = [
+    ('libunistring', '0.9.10'),
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-10.3.0.eb:24]:
```suggestion
dependencies = [
    ('ICU', '69.1'),
    ('libunistring', '0.9.10'),
]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-10.3.0.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-10.3.0.eb]: Done!

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.4, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/cc38cab66750e7af58f553f5e7866d2c for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Review by jfgrimm — APPROVED** [review]

**Comment by jfgrimm** [comment]
Going in, thanks @casparvl!

**Pull request merged**